test/assembly/DockerTest.java

```diff
@@ -48,11 +48,11 @@ public void bootup() throws InterruptedException, TimeoutException, IOException
         String imagePath = Paths.get("assemble-docker.tar").toAbsolutePath().toString();
         ProcessResult result = execute("docker", "load", "-i", imagePath);
         LOG.info(result.outputString());
-        // TODO: disable diagnostics reporting
         StartedProcess typeDBProcess = executor.command(
                 "docker", "run", "--name", "typedb",
                 "--rm", "-t", "-p", String.format("%d:%d", typeDBPort, typeDBPort),
-                "bazel:assemble-docker"
+                "bazel:assemble-docker",
+                "/opt/typedb-all-linux-x86_64/typedb", "server", "--diagnostics.reporting.enable=false"
```

Comment:
Override default docker command with custom one that also has arguments.

It might be better for the default Docker command to become an 'entrypoint', which means the `typedb server` part of the command are non-overridable.